rest-api/vip-endpoints.php

```diff
@@ -123,6 +131,94 @@ public function list_sites() {
 
 		return new WP_REST_Response( $sites );
 	}
+
+	/**
+	 * Build list of active plugins on site ()
+	 */
+	public function list_plugins() {
```

Comment:
As a best practice, it's good to keep API endpoint callbacks lean and without much logic, which means that they should call other functions to collect the data they need. The endpoint callback could be something as simple as:

```
function list_plugins() {
    plugins = get_all_the_plugins();
    if ( is_error( plugins ) ) {
        return error
    }

    return wp_rest_response
}
```

The nice thing about this approach is we can then re-use the `get_all_the_plugins` function in other contexts, if needed (e.g. CLI).